package.json

```diff
@@ -9,14 +9,15 @@
     "@fortawesome/free-solid-svg-icons": "6.1.0",
     "@fortawesome/react-fontawesome": "0.1.18",
     "@headlessui/react": "1.7.15",
-    "@itheum/sdk-mx-data-nft": "0.1.1",
+    "@itheum/sdk-mx-data-nft": "^0.1.1",
     "@multiversx/sdk-core": "12.8.0",
     "@multiversx/sdk-dapp": "2.20.1",
     "@multiversx/sdk-network-providers": "2.0.0",
-    "@radix-ui/react-dropdown-menu": "2.0.5",
-    "@radix-ui/react-navigation-menu": "1.1.3",
-    "@radix-ui/react-slot": "1.0.2",
-    "@radix-ui/react-switch": "1.0.3",
+    "@radix-ui/react-dialog": "^1.0.4",
+    "@radix-ui/react-dropdown-menu": "^2.0.5",
+    "@radix-ui/react-navigation-menu": "^1.1.3",
+    "@radix-ui/react-slot": "^1.0.2",
```

Comment:
Not good. We should use "1.0.2" instead of "^1.0.2". Not just for this lib, but for all of them

Reply:
Updated